spec/linter-spec.js

```diff
@@ -2,20 +2,29 @@
 
 import * as path from 'path';
 
+const travisYml = path.join(__dirname, '.travis.yml');
 const badPath = path.join(__dirname, 'files', 'bad.yaml');
 const issue2Path = path.join(__dirname, 'files', 'issue-2.yaml');
 const issue9Path = path.join(__dirname, 'files', 'issue-9.yaml');
 
 describe('Js-YAML provider for Linter', () => {
   const lint = require('../lib/linter-js-yaml.js').provideLinter().lint;
 
-  beforeEach(() =>
+  beforeEach(() => {
+    // Info about this beforeEach() implementation:
+    // https://github.com/AtomLinter/Meta/issues/15
+    const activationPromise = atom.packages.activatePackage('linter-js-yaml').then(() =>
+      atom.config.set('linter-js-yaml.customTags', ['!yaml', '!include'])
+    );
+
     waitsForPromise(() =>
-      atom.packages.activatePackage('linter-js-yaml').then(() =>
-        atom.config.set('linter-js-yaml.customTags', ['!yaml', '!include'])
-      )
-    )
-  );
+      atom.packages.activatePackage('language-yaml').then(() =>
+        atom.workspace.open(travisYml)
```

Comment:
Since no spec refers to this file, why not just open one of the files we already have a path to? The only important part is that the file be a YAML file to trigger the loading of the YAML language.